### ISSUE TITLE
Issue with timed Scheduler

### ISSUE THREAD

**Issue by CloudeLecaw** [Zrips/CMI#6771]
```
[20:00:00] [Server thread/WARN]: java.lang.NullPointerException: Cannot invoke "org.bukkit.entity.Player.getUniqueId()" because "<parameter3>" is null
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.process(SpecializedCommandManager.java:363)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.processCmds(SpecializedCommandManager.java:302)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.SpecializedCommands.SpecializedCommandManager.processCmds(SpecializedCommandManager.java:252)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.Sheduler.Schedule.perform(Schedule.java:391)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.Sheduler.Schedule.safePerform(Schedule.java:318)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.Sheduler.SchedulerManager.checkSchedulers(SchedulerManager.java:373)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.Sheduler.SchedulerManager.access$0(SchedulerManager.java:366)
[20:00:00] [Server thread/WARN]: 	at CMI_9.1.3.2.jar//com.Zrips.CMI.Modules.Sheduler.SchedulerManager$1.run(SchedulerManager.java:318)
[20:00:00] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftTask.run(CraftTask.java:101)
[20:00:00] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[20:00:00] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1531)
[20:00:00] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:483)
[20:00:00] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1455)
[20:00:00] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1225)
[20:00:00] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
[20:00:00] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
```

It happens when the scheduler shoudl trigger

```
    '13':
      Month: 3
      Day: 26
      Hour: 20
```
```
  Commands:
  - lottery draw
```

![image](https://user-images.githubusercontent.com/12080017/160259799-d3e08072-99d6-4a5d-845c-20c652369331.png)

**Comment by x1p** [comment]
Today, I found this out the hard way. Quite annoying error because I depend on this function for quite a lot of things.

However I found a workaround!

The problem is in the `delay! 5`
Qoute of the latest changelog:
> Small change for "delay! [amount]" variable, now it has **optional** delay name you can provide after delay itself, for example "delay! 5 serverStop"

This "optional" name is somehow not optional. If you provide it with a trivial name, it works. Or at least, in my case.
`delay! 5 bla`

**Comment by CloudeLecaw** [comment]
so all delays needs an optional name now?...

**Comment by x1p** [comment]
My schedules worked fine last night with an argument.

Short term: put an argument behind it, reload cmi and enjoy.
Long term: Zrips should fix it though

**Comment by CloudeLecaw** [comment]
oh now i see it also our custom aliases where we use

delay! x   <- so the format that was working all the time now shows this issues.. specialy when its executed from console :\

**Comment by x1p** [comment]
Seems to be fixed in CMI 9.1.3.3 although no mentioning in the changelog.

**Comment by CloudeLecaw** [comment]
I was able now to understand what happend. The placeholder was not updating in the scheduler utself well i solved it with let the scheduler trigger a secound diabled scheduler by command that then outputs the updated placeholder. Thank you for the suppport here and the ideas u have me on discord.
